src/main/java/io/github/_4drian3d/authmevelocity/lastserver/listener/PreSendOnLoginListener.java

```diff
@@ -25,13 +26,16 @@ public final class PreSendOnLoginListener implements LastLoginListener<PreSendOn
     @Override
     public EventTask executeAsync(final PreSendOnLoginEvent event) {
         return EventTask.withContinuation(continuation -> {
-            final String player = event.player().getUsername();
-            final String newServer = this.database.lastServerOf(player);
-            if (newServer != null) {
+            final Player player = event.player();
+            final String newServer = this.database.lastServerOf(player.getUsername());
+
+            Boolean getRequirePermission = plugin.config().get().getRequirePermission();
```

Comment:
It is not necessary to use the wrapper, just use the primitive type
```suggestion
            boolean getRequirePermission = plugin.config().get().getRequirePermission();
```